### PR TITLE
Bypass docker pull rate limit by overriding registries

### PR DESCRIPTION
**Pull request by Xenwar**
This PR introduces two variables used to override container registries.

### CODE REVIEW

File: jenkins/scripts/files/run_integration_tests.sh [hunk 1]
```diff
@@ -13,6 +13,11 @@ export IMAGE_OS
 export DEFAULT_HOSTS_MEMORY
 export NUM_NODES
 
+# Overrides docker.io
+export OVERRIDE_DOCKER_IO_REGISTRY=${OVERRIDE_DOCKER_IO_REGISTRY:-"registry.nordix.org/docker-hub-proxy/library"}
+# Overrides registry.hub.docker.com
+export OVERRIDE_DOCKER_HUB_REGISTRY=${OVERRIDE_DOCKER_HUB_REGISTRY:-"registry.nordix.org/docker-hub-proxy"}
```
Comment (fmuyassarov) [inline jenkins/scripts/files/run_integration_tests.sh:19]: Who is the consumer of this variable?

Comment (fmuyassarov) [reply jenkins/scripts/files/run_integration_tests.sh:19]: `OVERRIDE_DOCKER_HUB_REGISTRY`

Comment (Xenwar) [reply jenkins/scripts/files/run_integration_tests.sh:19]: other repos:
one example is https://github.com/metal3-io/cluster-api-provider-metal3/pull/155/files

**Comment by fmuyassarov** [comment]
We have fixed this in our image building process. 
/close

**Pull request closed without merging**